feg/cloud/go/services/feg/obsidian/models/swagger.v1.yml

```diff
@@ -1220,6 +1226,19 @@ definitions:
         example: ":56789"
         x-nullable: false
 
+  virtual_apn_rule:
+    description: Virtual APN Rule configuration
+    type: object
+    properties:
+      apn_filter:
```

Comment:
Consider adding descriptions for the individual properties as well. Just for extra clarity. :)